packages/@aws-cdk/aws-ecs/test/aws-log-driver.test.ts

```diff
@@ -155,4 +155,41 @@ describe('aws log driver', () => {
 
 
   });
+
+  test('allows cross-region log group', () => {
+    // GIVEN
+    const logGroupRegion = 'asghard';
+    const logGroup = logs.LogGroup.fromLogGroupArn(
+      stack,
+      'LogGroup',
+      `arn:aws:logs:${logGroupRegion}:1234:log-group:my_log_group`,
+    );
```

Comment:
```suggestion
    const logGroup = logs.LogGroup.fromLogGroupArn(stack, 'LogGroup',
      `arn:aws:logs:${logGroupRegion}:1234:log-group:my_log_group`);
```

Reply:
Yeah, I noticed you prefer that style in cdk. I am more accustomed to Python and having everything spread across multiple lines. Makes it much easier to follow `git diff`